src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1328,6 +1330,51 @@ void *instanceservicesession(void *dummyPt) {
             std::string memoryUsage = JasmineGraphInstanceService::requestPerformanceStatistics(
                 isVMStatManager, isResourceAllocationRequired);
             write(connFd, memoryUsage.c_str(), memoryUsage.size());
+        } else if (line.compare(JasmineGraphInstanceProtocol::START_STAT_COLLECTION)  == 0) {
+            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::START_STAT_COLLECTION, "info");
+            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
+            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
+            collectValid=true;
```

Comment:
Need to maintain spaces between assignments. Please check all the newly added lines and fix the issue if exists.

Reply:
Changes are done

Reply:
Ok